doc/object_wrap.md

```diff
@@ -190,25 +190,16 @@ property of the `Napi::CallbackInfo`.
 
 Returns a `Napi::Function` representing the constructor function for the class.
 
-### OverrideFinalizeCallback
+### Finalize
 
-Overrides default `Napi::ObjectWrap::FinalizeCallback` with a user defined finalizer.
+Hooks into `Napi::ObjectWrap::FinalizeCallback` giving access to `Napi::Env`
```

Comment:
How about "This method is called immediately before before the wrapped native object instance is freed. Overriding it provides an opportunity to run cleanup code that requires access to the `Napi::Env`."?

Reply:
Love it!

Do we need to rearrange the sentence so it starts with a verb to match the pattern with the other methods documentation?

For example: Provides an opportunity to run cleanup code that requires access to the `Napi::Env` before the wrapped native object instance is freed.  Override to implement.

Reply:
SGTM